config/kubernetes-csi/org.yaml

```diff
@@ -327,3 +327,20 @@ teams:
     - lpabon
     - saad-ali
     privacy: closed
+  csi-misc:
+    description: Miscellaneous Discussions for Kubernetes CSI Work Group
+    maintainers:
+    - childsb
+    - msau42
+    - saad-ali
+    members:
+    - chakri-nelluri
+    - davidz627
+    - gnufied
+    - jsafrane
+    - lpabon
+    - pohly
+    - rootfs
+    - sbezverk
+    - vladimirvivien
```

Comment:
Can you add me?  xing-yang

Reply:
I tried:

```
    config_test.go:248: The following members of team csi-misc are not Kubernetes CSI org members: xing-yang
```

Reply:
Oh, csi-org, I can fix that easily here.

Reply:
Maybe I need to apply for kubernetes-csi org membership first?  I should do that anyway.

Reply:
@xing-yang -- You'll need to [apply for membership for the `kubernetes-csi` org](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E) before @jsafrane can add you to this GitHub team.

Reply:
Fixed.

Reply:
My membership application is approved and I'm added to kubernetes-csi org now.

Reply:
Thanks for sorting that! :)